.taskcluster.yml

```diff
@@ -43,11 +43,13 @@ tasks:
                rustup component add clippy rustfmt &&
                rustup toolchain install nightly &&
                curl -L https://github.com/est31/cargo-udeps/releases/download/v0.1.25/cargo-udeps-v0.1.25-x86_64-unknown-linux-gnu.tar.gz | tar xz -C /usr/local/cargo/bin --strip-components 2 &&
+               curl -L https://github.com/rustsec/rustsec/releases/download/cargo-audit%2Fv0.16.0/cargo-audit-x86_64-unknown-linux-musl-v0.16.0.tgz | tar xzv -C /usr/local/cargo/bin --strip-components 1 &&
                git clone --quiet ${repository} &&
                cd rust-code-analysis &&
                git -c advice.detachedHead=false checkout ${head_rev} &&
                pip3 install --quiet pre-commit &&
                pre-commit run -a --show-diff-on-failure &&
+               pre-commit run --show-diff-on-failure  -c .pre-commit-audit-config.yaml --files Cargo.lock Cargo.toml &&
```

Comment:
Add a comment saying we have a separate config because we only want to run it when dependencies are touched